tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,699 @@
+	helm.Install(GinkgoT(), helmOptions, clusterChartPath, releaseName)
+
+	// Wait for cass-operator pod to be ready
+	attempts := 0
+	maxAttempts := 10
+	for {
+		attempts++
+		clientset, _ := k8s.GetKubernetesClientFromOptionsE(GinkgoT(), kubectlOptions)
+		pods, _ := clientset.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app.kubernetes.io/name=cass-operator"})
+		if len(pods.Items) == 1 {
+			k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "pod", "-l", "app.kubernetes.io/name=cass-operator", "--timeout=1800s")
+			break
+		} else if attempts > maxAttempts {
+			Fail("Couldn't find cass-operator pod")
+		}
+		time.Sleep(20 * time.Second)
+	}
+
+	// Wait for CassandraDatacenter to be ready..
+	k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "cassandradatacenter/dc1", "--timeout=1800s")
+}
+
+func GetUsernamePassword(secretName, ns string) credentials {
+	kubectlOptions := k8s.NewKubectlOptions("", "", ns)
+	secret := k8s.GetSecret(GinkgoT(), kubectlOptions, secretName)
+	username := secret.Data["username"]
+	password := secret.Data["password"]
+	creds := credentials{string(username), string(password)}
+	return creds
+}
+
+func runCassandraQueryAndGetOutput(query string) string {
+	cqlCredentials := GetUsernamePassword("k8ssandra-superuser", namespace)
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+	// Get reaper service
+	output, _ := k8s.RunKubectlAndGetOutputE(GinkgoT(), kubectlOptions, "exec", "-it", "k8ssandra-dc1-default-sts-0", "--", "/opt/cassandra/bin/cqlsh", "--username", cqlCredentials.username, "--password", cqlCredentials.password, "-e", query)
+	return output
+}
+
+func WaitForPodWithLabelToBeReady(label string, waitTime time.Duration, maxAttempts int) {
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+	attempts := 0
+	for {
+		attempts++
+		getCassandraPodOutput, err := k8s.RunKubectlAndGetOutputE(GinkgoT(), kubectlOptions, "get", "pods", "-l", label)
+		if err == nil && !strings.HasPrefix(getCassandraPodOutput, "No resources found") {
+			break
+		}
+		if attempts > maxAttempts {
+			Fail(fmt.Sprintf("Pod with label '%s' didn't start within timeout", label))
+		}
+		time.Sleep(waitTime)
+	}
+	k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "pod", "-l", label, "--timeout=1800s")
+}
+
+func GetStargateService() v1.Service {
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+	clientset, err := k8s.GetKubernetesClientFromOptionsE(GinkgoT(), kubectlOptions)
+	if err != nil {
+		Fail("Couldn't get k8s client")
+	}
+
+	services, err := clientset.CoreV1().Services(namespace).List(context.Background(), metav1.ListOptions{})
+	for _, service := range services.Items {
+		if strings.HasSuffix(service.ObjectMeta.Name, "-stargate-service") {
+			return service
+		}
+	}
+	Fail(fmt.Sprintf("Couldn't find the Stargate service"))
+	return v1.Service{}
+}
+
+var (
+	Info    = Yellow
```

Comment:
Are these supposed to be mutable strings or enums or what?

Reply:
Copy/pasted this from a webpage after searching for ways to print colored log lines.
I'd go with enums here as they don't need to mutate.
I guess I should use `const` instead then, right?

Reply:
With enums you would use iota.